linux/shim/client.go

```diff
@@ -138,9 +158,10 @@ func WithLocal(ctx context.Context, config Config) (shim.ShimClient, io.Closer,
 }
 
 type Config struct {
-	Address   string
-	Path      string
-	Namespace string
+	Address    string
+	Path       string
+	Namespace  string
+	CgroupPath string
```

Comment:
The cgroup is created and configured by the client.
Containerd is only responsible for creating containerd-shim inside the cgroup.